compiler/src/utils/config.re

```diff
@@ -624,3 +624,6 @@ let get_implicit_opens = () => {
     ]);
   };
 };
+
+let format_in_place =
```

Comment:
@marcusroberts @ospencer Should we make an issue to split out the grainformat config items into a module that's not in the compiler? It feels odd to have it here

Reply:
Somewhat related—I hadn't realized that the tools inherit all of the compiler config. We should probably also add to that issue (or make an issue for that too). It seems like they only need the stdlib path and include paths to be able to enter the typechecking phase, but I think they should just take those flags and ignore the rest.

Reply:
#917 created. Feel free to edit with any clarifications you think are necessary